### PR TITLE
splits events/<event-id>/speakers into proper pages

### LINKED ISSUE #245: enhancement: split events/<event-id>/speakers into proper pages
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

split all the sub routes under `events/<event-id>/speakers` into proper pages using dynamic segments.

image

`events/<event-id>/speakers`
`events/<event-id>/speakers/pending`
`events/<event-id>/speakers/accepted`
`events/<event-id>/speakers/rejected`

All pages should be done via dynamic route segments similar to #222 .

### PR DESCRIPTION
**Pull request by anu-007**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
splits events//sessions into proper pages

#### Changes proposed in this pull request:
image

demo- https://open-event-frontend-branch4.herokuapp.com/events/963d3ba5/speakers/rejected

Fixes #245

### CODE REVIEW

File: app/templates/events/view/speakers/list.hbs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          <td>
+            <div class="ui ordered list">
+              {{#each speaker.submitted_session as |session|}}
+                <div class="item">{{session}}</div>
+              {{/each}}
+            </div>
+          </td>
+          <td>
+            <div>
+              {{#if (eq speaker.state "confirmed")}}
```
Comment (niranjan94) [inline app/templates/events/view/speakers/list.hbs]: Single quotes please.

**Comment by anu-007** [comment]
@niranjan94 please review.thanks

**Comment by anu-007** [comment]
@niranjan94  @geekyd @CosmicCoder96 please review.

**Comment by anu-007** [comment]
@niranjan94 @geekyd updated branch please merge.thanks

**Comment by anu-007** [comment]
@niranjan94 @geekyd please approve and merge .thanks!!

**Pull request merged**